web/src/user_group_edit.ts

```diff
@@ -1283,6 +1300,73 @@ export function remove_deactivated_user_from_all_groups(user_id: number): void {
     }
 }
 
+export function update_displayed_groups(filter_id: string | undefined): void {
+    assert(filter_id !== undefined);
```

Comment:
If there is an assert here, the type of `filter_id` should be just `string`.

Reply:
Now there is no need of assert here. In the previous comment, I meant that if we have an assert then we are sure that the variable is string, so there is no need of assert statement or `undefined` in type.